src/components/WebexRemoteMedia/WebexRemoteMedia.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {useMeeting, useVideo} from '../hooks';
+import {WebexAvatar} from '../';
+
+import './WebexRemoteMedia.scss';
+
+/**
+ * Webex Remote Media component displays the user's remote video
+ *
+ * @param {object} props
+ * @returns {object} JSX of the component
+ */
+export default function WebexRemoteMedia({meetingID, personID}) {
+  const {remoteVideo} = useMeeting(meetingID);
+  const videoRef = useVideo(remoteVideo);
+
+  return (
+    <div className="remote-media">
+      {remoteVideo ? (
+        <>
+          <video ref={videoRef} playsInline />
+          <audio ref={videoRef} playsInline />
+        </>
+      ) : (
+        <WebexAvatar personID={personID} />
```

Comment:
If there is no remote stream (e.g. network error), I don't think we should be displaying the user's avatar